lib/database.coffee

```diff
@@ -267,45 +267,42 @@ module.exports = (env) ->
         table.boolean('discrete')
         table.timestamp('lastUpdate').nullable()
         table.string('lastValue').nullable()
+        table.index(['deviceId','attributeName'], 'deviceAttributeDeviceIdAttributeName')
+        table.index(['deviceId'], 'deviceAttributeDeviceId')
+        table.index(['attributeName'], 'deviceAttributeAttributeName')
       )
 
-      # add to old deviceAttribute table 
-      pending.push @knex.schema.table('deviceAttribute', (table) =>
-        table.boolean('discrete').nullable()
-      ).catch( (error) -> 
-        if error.errno is 1 then return #ignore
-        throw error
+      @knex.schema.hasTable('deviceAttribute').then( (tableExists) =>
```

Comment:
Thanks for the pull request. Could you delete the whole code to add the `discrete` column, if it does not exist. I think we don't need this code anymore, because nobody uses a old database that hasn't this column anymore. Thanks.